examples/BasicSerialLogger/BasicSerialLogger.ino

```diff
@@ -19,30 +25,23 @@ const char globalProgmemMessageWithRamString[] PROGMEM
 
 // Need to remember that backslash must be escaped in the string text
 // Banner generated with https://devops.datenkollektiv.de/banner.txt/index.html
-const char banner0[] PROGMEM = " ___   ___   ___   _ _    _                 ";
-const char banner1[] PROGMEM = "| _ \\ | __| |_  ) | | |  | |     ___   __ _ ";
-const char banner2[] PROGMEM = "|   / | _|   / /  |_  _| | |__  / _ \\ / _` |";
-const char banner3[] PROGMEM = "|_|_\\ |_|   /___|   |_|  |____| \\___/ \\__, |";
-const char banner4[] PROGMEM = "                                      |___/ ";
+const PROGMEM char banner[5][]  = { "   ___    ____   ___   ____   __             ",
+                                    "  / _ \\  / __/  |_  | / / /  / /  ___   ___ _",
+                                    " / , _/ / _/   / __/ /_  _/ / /__/ _ \\ / _ `/",
+                                    "/_/|_| /_/    /____/  /_/  /____/\\___/ \\_, / ",
+                                    "                                      /___/  "}

```

Comment:
A semicolon missing at the end.

Reply:
This is probably my bad. Sometimes I forget about the semicolon in C++ (especially coming from a python day of coding).

I took the liberty to replace the ASCII art on my dev-docs branch (the commits you merged), trying to reduce the amount of bytes used in the sketch.

Reply:
No problem. Compiler shows error. That was easy to fix :)